### DISCUSSION TITLE
How to hide collectionId and collectionName fields from API?

### DISCUSSION THREAD

**Discussion by unkn0wncode** [pocketbase/pocketbase#5532] [Q&A] [answered]
Hey folks, I don't seem to be able to find this question in the google, so thought to just ask.
How to hide fields like:
```
collectionId
collectionName
```
from being displayed in the API? Should I manually strip them somehow? They seem to appear in basically every output from the database and it feels weird to display it publicly.
Is there a switch for that that I missed?

Thank you!

**Comment by ganigeorgiev** [comment]
There is no builtin option to hide the system fields.

> it feels weird to display it publicly

You don't have to display them and can just ignore them.

**Reply by unkn0wncode** [reply]
> There is no builtin option to hide the system fields.
> 
> > it feels weird to display it publicly
> 
> You don't have to display them and can just ignore them.

I understand. I guess while I can ignore displaying them in parts of the website, they still will be visible in the network tab for whoever wants to see, unless i'd strip them manually.
Is hiding the option possibly planned in the future?

Thank you for the quick reply!

**Reply by ganigeorgiev** [reply, marked as answer]
> they still will be visible in the network tab for whoever wants to see

I don't see why this is a problem.

> Is hiding the option possibly planned in the future?

There will be a new hook and helper `record.Hide()` method with the upcoming v0.23.0 release (see an example in https://github.com/pocketbase/pocketbase/discussions/5369#discussioncomment-10524248) but there will be no option for doing this from the Dashboard.
